plugins/community/scalapb-scala/v0.11.12/Dockerfile

```
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1.4
+FROM debian:bullseye-20221114 AS build
```

When adding new versions of a plugin (or a new plugin entirely), we start with the latest version of base images.

```suggestion
FROM debian:bullseye-20230109 AS build
```